content/en/actions/forms/_index.md

```diff
@@ -0,0 +1,93 @@
+disable_toc: false
+---
+
+{{< callout url="LINK GOES HERE" btn_hidden="false" header="Join the Preview!" >}}
+Agents are in Preview. Click <b>Request Access</b> and fill in the Datadog Product Preview Program form.
+{{< /callout >}}
+
+## Overview
+
+Datadog Forms allows you to collect input, analyze responses, and trigger automations in Datadog. Forms and their responses can be shared across your organization, allowing you to collect and analyze data with your team. Because you don't need a Datadog account to respond to forms, they can also be shared with customers, internal teams, and other stakeholders. 
+
+## Examples
+
+Here are some ways you can use forms:  
+- create a GitHub repository from a template using [our scaffolding blueprint][3]. 
+- run developer surveys in an internal developer portal (IDP).
+- collect employee feedback about security concerns through a form that creates a case.
+
+## Create a form
+
+To create a new form: 
+1. Navigate to the [Forms][1] page.
+1. Click **+ New Form**.
+1. New forms are auto-populated with placeholder component. To edit the form, click the placeholder component, or click the **<i class="icon-plus-circled-wui"></i>** icon to add a new component. Component types include single line text, multi-line text, select, checkboxes, and ratings. The following table lists the elements available inside components:
+    | Element  | Description    | Component Availability |
+    |----------------------------|---------------------------------------------------------------------|-----|
+    | Required | Determines whether the question requires an answer from the respondent; required questions are marked with a red asterisk on forms. | All components | 
+    | Field name           | The name of the field; appears in the JSON version in [**Responses**][8]; not visible to respondents.                                   | All components | 
+    | Placeholder         | The text the respondent sees before entering any text.                 | All components except ratings |
+    | Default value         | The default item or text that is selected before the respondent makes a selection.                                | All components except ratings | 
+    | Data         | The available options that respondents can choose from.                | Select and checkboxes | 
+    | Questions             | The questions respondents are asked when rating their experience. | Ratings |  
+1. Click **Save** to save your changes. 
+1. Click **View** to display the form as it appears to respondents. Click **Edit** to return to the creator view. 
+1. Click **Share** to copy the form link to your clipboard.
+1. In the **Debug** section:
+    1. click **Form** to show a mini-preview of your form.
+    1. click **Data Definition** to show the JSON version of the components' definition.
+    1. click **UI Definition** to show the JSON version of the form's UI.
```

Comment:
Since these steps are optional, I would put them separate from the main procedure. Perhaps come up with a sentence that describes them, and put them in a bulleted list.